codegen/src/test/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeSpecTest.java

```diff
@@ -199,6 +206,14 @@ static class TestCase {
         private final String outputFileSuffix;
         private final String caseName;
 
+
+        @Override
+        public String toString() {
```

Comment:
This is needed to get the Test case names in the Parameterized test case titles for each tests